docs/HowToUpgrade.md

```diff
@@ -174,18 +174,18 @@ helm upgrade -n nuodb database nuodb/database \
 
 > **Note**: In multi-cluster deployments, it is enough to enable the feature on the entry point cluster _only_.
 
-To confirm that the database protocol has been upgraded using `nuocmd show database-versions` and check the database effective version.
-There should be _no_ protocol versions in the available versions section.
+To verify the database protocol upgrade, run `nuocmd show database-versions` and check the effective database versions in the response.
+The `Available versions` must be empty.
 
-A random TE will be automatically restarted after upgrading the protocol.
+After upgrading the protocol one of the TE will be automatically restarted.
```

Comment:
Nit: "... one of the TE" -> "... one of the TEs"

Reply:
Thanks